README.md

```diff
@@ -1,45 +1,55 @@
 # AIP++
 
-The full documentation of our application can be found at 
-https://cutt.ly/Mn78T3f
-### Running the application locally using Docker
+> A Software Tool for Finding Related Work in Academia
 
+<ins>A</ins>rticle <ins>I</ins>nformation <ins>P</ins>arser is an instrument to parse, unify, and in some cases correct article metadata. AIP creates a PostgreSQL database that allows for easily finding related work. AIP++ is an extension of the original tool to improve on the article discovery and querying capabilities of the original tool.
 
-1. Make sure you have Docker installed and running on your machine.
+More information on how this tool works can be found in the [project report](https://drive.google.com/file/d/1OUJvakiJYdlw-jV2ZKjDs5ggCNi2DyiW/view).
 
-2. Run 'docker-compose up' command in the root directory of the project.
+![Web UI screenshot](docs/images/web-ui-screenshot.png)
 
-3. Access the application through your web browser by going to 
-'http://localhost:8000/'.
-   
-### Accessing the application remotely
+## Setup
+
+### Prerequisites
+
+:fire: Please read this section before continuing :fire:
+
+In order to run the following application, make sure that you have [Docker](https://www.docker.com/products/docker-desktop).
 
-The most recent version of the application can be accessed at 
-https://aip.irqize.dev/
+It is also important that you have a recent version of the database which can be obtained by contacting us directly. You can also generate your own database by following the instructions [here](backend/automatic_checks/automatic_check_readme.md).
```

Comment:
How is the database maintained at the moment? Do we know who is _us_ and how to get a copy of the database?

Reply:
At the moment, I have a working version locally, but not all features work as I've been trying AIP out more, so I'm not sure where we could put the current working version for the time being. As for the contact point, _us_ in this case is me for the moment, but eventually we should have a way to get the latest version (with more articles, etc.). However, that is going to take a bit more effort. Would it be an option to host a copy of my version on the At Large website server? As is done for the current dump: https://atlarge-research.com/data/2020_fgcs_aip.pgsql

Reply:
Sure, let's discuss over Slack how to get that done.

Is it useful to have a pre-built database available if users can build it themselves easily?